### PR TITLE
(grc_conversion) Epidoc and CTS conversion

### PR DESCRIPTION
**Pull request by lcerrato**
#1399 tlg0028 translation batch

### CODE REVIEW

File: data/tlg0028/tlg002/tlg0028.tlg002.perseus-grc2.xml [hunk 1]
```diff
@@ -36,14 +36,14 @@
...
-				<editor>K. J. Maidment</editor>
+				<editor>Kenneth John Maidment</editor>
 				<imprint>
 					<pubPlace>London</pubPlace>
 					<publisher>William Heinemann, Ltd.</publisher>
 					<pubPlace>Cambridge, MA</pubPlace>
 					<publisher>Harvard University Press</publisher>
-					<date>1941</date>
-					<date type="printing">1960</date>
+					<date type="printng">1941</date>
```
Comment (AlisonBabeu) [inline data/tlg0028/tlg002/tlg0028.tlg002.perseus-grc2.xml:45]: Typo here.

File: data/tlg0028/tlg003/tlg0028.tlg003.perseus-eng2.xml [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+          <monogr>
+            <author>Andocides</author>
+            <title>Minor Attic Orators</title>
+            <editor>Kenneth John Maidment</editor>
+            <imprint>
+              <pubPlace>London</pubPlace>
+              <publisher>William Heinemann, Ltd.</publisher>
+              <pubPlace>Cambridge, MA</pubPlace>
+              <publisher>Harvard University Press</publisher>
+              <date type="printng">1941</date>
```
Comment (AlisonBabeu) [inline data/tlg0028/tlg003/tlg0028.tlg003.perseus-eng2.xml:44]: Typo.

File: data/tlg0028/tlg004/tlg0028.tlg004.perseus-eng2.xml [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+          <monogr>
+            <author>Andocides</author>
+            <title>Minor Attic Orators</title>
+            <editor>Kenneth John Maidment</editor>
+            <imprint>
+              <pubPlace>London</pubPlace>
+              <publisher>William Heinemann, Ltd.</publisher>
+              <pubPlace>Cambridge, MA</pubPlace>
+              <publisher>Harvard University Press</publisher>
+              <date type="printng">1941</date>
```
Comment (AlisonBabeu) [inline data/tlg0028/tlg004/tlg0028.tlg004.perseus-eng2.xml:44]: Typo.

File: data/tlg0028/tlg004/tlg0028.tlg004.perseus-grc2.xml [hunk 4]
```diff
@@ -36,14 +36,14 @@
...
-<editor>K. J. Maidment</editor>
+<editor>Kenneth John Maidment</editor>
 <imprint>
 <pubPlace>London</pubPlace>
 <publisher>William Heinemann, Ltd.</publisher>
 <pubPlace>Cambridge, MA</pubPlace>
 <publisher>Harvard University Press</publisher>
-<date>1941</date>
-<date type="printing">1960</date>
+<date type="printng">1941</date>
```
Comment (AlisonBabeu) [inline data/tlg0028/tlg004/tlg0028.tlg004.perseus-grc2.xml:45]: Typo.

File: data/tlg0028/tlg006/tlg0028.tlg006.perseus-eng2.xml [hunk 5]
```diff
@@ -0,0 +1,247 @@
...
+          <monogr>
+            <author>Andocides</author>
+            <title>Minor Attic Orators</title>
+            <editor>Kenneth John Maidment</editor>
+            <imprint>
+              <pubPlace>London</pubPlace>
+              <publisher>William Heinemann, Ltd.</publisher>
+              <pubPlace>Cambridge, MA</pubPlace>
+              <publisher>Harvard University Press</publisher>
+              <date type="printng">1941</date>
```
Comment (AlisonBabeu) [inline data/tlg0028/tlg006/tlg0028.tlg006.perseus-eng2.xml:42]: Typo

**Review by AlisonBabeu — CHANGES_REQUESTED** [review]
Looks good, other than the typo "printng" rather than "printing" in a few places.

**Comment by lcerrato** [comment]
shocked that there is only one...

**Comment by lcerrato** [comment]
@AlisonBabeu  Did I get them all?

**Pull request merged**